pyston/data_processor.py

```diff
@@ -94,21 +94,18 @@ def _validate_not_empty(self, data_item, key, item):
     def _get_mimetype_from_filename(self, filename):
         return mimetypes.types_map.get('.{}'.format(filename.split('.')[-1])) if '.' in filename else None
 
-    def _get_content_type(self, data_item, filename):
-        return data_item.get('content_type') or self._get_mimetype_from_filename(filename)
-
     def _process_file_data(self, data, files, key, data_item, file_content):
         filename = data_item.get('filename')
-        content_type = self._get_content_type(data_item, filename)
-        if content_type:
+        mimetype = self._get_mimetype_from_filename(filename)
+        if not mimetype:
+            self.errors[key] = ugettext('Invalid file name')
```

Comment:
tady bych to ještě zabalil

self.errors[key] = RESTDictError({'filename': ugettext('Invalid suffix of the file')})